### PR TITLE
app-admin/mkosi: bump to v15.1

### PR DESCRIPTION
**Pull request by 257**

### CODE REVIEW

File: app-admin/mkosi/metadata.xml [hunk 1]
```diff
@@ -10,6 +10,6 @@
 		<name>Proxy Maintainers</name>
 	</maintainer>
 	<upstream>
-		<remote-id type="github">systemd/mkosi</remote-id>
+		<remote-id type="github">webcamoid/webcamoid</remote-id>
```
Comment (Flowdalic) [inline app-admin/mkosi/metadata.xml:13]: That looks wrong.

**Comment by Flowdalic** [comment]
Latest version is 17.1. Could you bump to that?

File: app-admin/mkosi/metadata.xml (hunk 1 above)
Comment (257) [reply app-admin/mkosi/metadata.xml]: it is. good catch, thanks.

**Comment by 257** [comment]
i have not looked into gentoo bits of mkosi after 15.1 and upstream stopped touching that. lets go with 15.1 for now; i will look into 17.1 in the next round.

File: app-admin/mkosi/mkosi-15.1.ebuild [hunk 2]
```diff
@@ -5,23 +5,20 @@ EAPI=8
...
 DESCRIPTION="Build Bespoke OS Images"
 HOMEPAGE="https://github.com/systemd/mkosi"
-SRC_URI="https://github.com/systemd/mkosi/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
-S="${WORKDIR}/${PN}-${COMMIT}"
+SRC_URI="https://github.com/systemd/mkosi/archive/refs/tags/${RELEASE}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="GPL-2+"
 SLOT="0"
-KEYWORDS="~amd64"
+KEYWORDS="amd64"
```
Comment (dwfreed) [inline app-admin/mkosi/mkosi-15.1.ebuild:18]: This is a straight-to-stable...

**Review by dwfreed — CHANGES_REQUESTED** [review]

**Comment by 257** [comment]
oops! fixed.

**Comment by Flowdalic** [comment]
Your fixed of the first commit went into the second commit. Furthermore, I'd like to keep mkosi 14 as a future stable version. Please drop the drop commit. :)

**Pull request closed without merging**